protocols/direct-fund/direct-fund.go

```diff
@@ -228,7 +228,8 @@ func (s DirectFundObjective) safeToDeposit() bool {
 		chainHolding, ok := s.C.OnChainFunding[asset]
 
 		if !ok {
-			return false
+			// If there are no holdings on chain we use 0 for our calculations
+			chainHolding = big.NewInt(0)
```

Comment:
Does this change belong on this PR?

Reply:
This has been pulled out into https://github.com/statechannels/go-nitro/pull/215